src/Core/Domain/Cart/Command/UpdateCartDeliverySettingsCommand.php

```diff
@@ -43,14 +40,32 @@ class SetFreeShippingToCartCommand
      */
     private $allowFreeShipping;
 
+    /**
+     * @var bool
+     */
+    private $isAGift;
+
+    /**
+     * @var bool
+     */
+    private $useRecycledPackaging;
```

Comment:
I don't see the reason to use private if the class is not final.
Use protected, otherwise, no one will be able to extend it, and if the extension is forbidden, use final :)

Reply:
This is a Command. I'm not against starting to make all our Command properties `protected` but I think it should be a global change (maybe directed by an ADR ?) rather than just these properties.

And this PR aims to fix a bug blocking 1.7.7.0 RC1 build 2 😅 I dont think it's the right moment to start discussing such changes.

Can we move the global discussion "should all our Command properties be `protected`" for 1.7.8 😉 ?

Reply:
It's not only the ability to change the private variables, but it's also a usage problem.
I agree to have an issue about this problem :)

Reply:
See https://github.com/PrestaShop/PrestaShop/issues/21383